packages/frontend/amp/components/Body.tsx

```diff
@@ -5,17 +5,38 @@ import { css } from 'emotion';
 import { ArticleModel } from '@frontend/amp/pages/Article';
 import { TopMeta } from '@frontend/amp/components/TopMeta';
 import { SubMeta } from '@frontend/amp/components/SubMeta';
+import { pillarPalette } from '../../lib/pillars';
 
-const body = css`
+const body = (pillar: Pillar) => css`
     background-color: white;
+
+    ${bulletStyle(pillar)}
+`;
+
+const bulletStyle = (pillar: Pillar) => css`
```

Comment:
We've talked about this IRL, but I wonder whether there are better places for this. Placing styles with selectors on high-level components like Body breaks our CSS encapsulation/reintroduces semi-global CSS.

Instead what about:

* moving the shared styles into a separate file and importing that to allow for re-use
* duplicating the styles in the low-level components

(My preference is for the first.)

Agree this is something the design system should solve though, just it doesn't exist yet.

Reply:
Aside from design system, dcr needs to figure out the desired css architecture.

Reply:
I agree this pattern is not ideal and is something we need to work out how to avoid in the future. The `ArticleBody` component in web is also over-reliant on semi-global styles.

Off the top of my head, there's at least one more suggestion in addition to @nicl's ideas:

- provide a limited number of global / utility classes

Although I'm not sure how we would enforce the "limited" aspect of this. It would also add more cognitive overhead, as developers would have one more place to check where styles are coming from.

Reply:
Let's figure this out [outside of this PR](https://trello.com/c/jVeLpVUM/467-shared-styles)

Reply:
I agree with these comments but would like to defer fixing this until we agree the client side architecture.